### PR TITLE
.gitmodules: update branches

### PR DESCRIPTION
**Pull request by alexgg**
The branches are used by renovate when updating submodules to the latest branch head.

Changelog-entry: update git submodule branches

### CODE REVIEW

**Comment by acostach** [comment]
lgtm

File: .gitmodules [hunk 1]
```diff
@@ -17,19 +17,19 @@
 [submodule "layers/meta-freescale"]
 	path = layers/meta-freescale
 	url = https://github.com/Freescale/meta-freescale.git
-	branch = honister
+	branch = kirkstone
 [submodule "layers/meta-freescale-3rdparty"]
 	path = layers/meta-freescale-3rdparty
 	url = https://github.com/Freescale/meta-freescale-3rdparty.git
-	branch = honister
+	branch = master
```
Comment (floion) [inline .gitmodules:24]: Do we want `Kirkstone` here also?

**Comment by alexgg** [comment]
@resin-jenkins retest this please

**Comment by alexgg** [comment]
Non-functional change - will just make sure renovate updates submodule branches.

**Pull request merged**